src/h5web/visualizations/index.tsx

```diff
@@ -110,16 +110,11 @@ export const VIS_DEFS: Record<Vis, VisDef> = {
     Toolbar: NxSpectrumToolbar,
     Container: NxSpectrumContainer,
     supportsEntity: (entity) => {
-      if (!isGroup(entity)) {
+      if (!isGroup(entity) || !isNxDataGroup(entity)) {
```

Comment:
This is all that's needed now! Note that there's already a test for this use case.